addons/l10n_ch/models/res_bank.py

```diff
@@ -10,32 +10,9 @@
 from odoo.exceptions import ValidationError, UserError
 from odoo.tools.misc import mod10r
 
-ISR_SUBSCRIPTION_CODE = {'CHF': '01', 'EUR': '03'}
+QR_SUBSCRIPTION_CODE = {'CHF': '01', 'EUR': '03'}
 CLEARING = "09000"
```

Comment:
This postfinance cleaning can be removed.

With QR-bill it doesn't make anymore a difference if the bank account is bank or postal. Both uses IBAN and we don't use the postal account number anymore.